prefect_aws/credentials.py

```diff
@@ -1,15 +1,13 @@
 """Module handling AWS credentials"""
-
-from dataclasses import dataclass
-from typing import Optional
-
 import boto3
+from prefect.blocks.core import Block
+from pydantic import SecretStr
+from typing import Optional
```

Comment:
I think these imports might be out of order if I am not mistaken. 

Inside your repo directory, `prefect-aws`, can you try running
```
pre-commit install
pre-commit run --all
```
And see if there are any errors?

Reply:
It should fix the static analyses
https://github.com/PrefectHQ/prefect-aws/runs/7509687176?check_suite_focus=true